.github/workflows/docker.yml

```diff
@@ -0,0 +1,38 @@
+name: Docker test and publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Ruby 2.6
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.6.x
+      - name: Lint container
+        working-directory: docker
+        run: |
+          make lint
+      - name: Build container
+        env:
+          PUPPERWARE_ANALYTICS_STREAM: production
+          IS_LATEST: true
+        working-directory: docker
+        run: |
+          gem install bundler
```

Comment:
This could go with the "setup ruby" step so we have our Ruby environment complete and ready-to-go for all subsequent steps.

Not a big deal, just thinking about organization/cohesion.

Reply:
Updated. It has to be its own step since you can't have `run` and `uses` in the same step, but moved it higher up at least